src/main/kotlin/no/nav/k9/los/domene/repository/OppgaveRepository.kt

```diff
@@ -34,57 +32,64 @@ class OppgaveRepository(
     private val refreshOppgave: Channel<UUID>
 ) {
     private val log: Logger = LoggerFactory.getLogger(OppgaveRepository::class.java)
-    fun hent(): List<Oppgave> {
-        var spørring = System.currentTimeMillis()
-        val json: List<String> = using(sessionOf(dataSource)) {
-            it.run(
-                queryOf(
-                    "select data from oppgave",
-                    mapOf()
-                )
-                    .map { row ->
-                        row.string("data")
-                    }.asList
-            )
-        }
-        spørring = System.currentTimeMillis() - spørring
-        val serialisering = System.currentTimeMillis()
-        val list = json.map { s -> objectMapper().readValue(s, Oppgave::class.java) }.toList()
-        Databasekall.map.computeIfAbsent(object {}.javaClass.name + object {}.javaClass.enclosingMethod.name) { LongAdder() }
-            .increment()
 
-        log.info("Henter: " + list.size + " oppgaver" + " serialisering: " + (System.currentTimeMillis() - serialisering) + " spørring: " + spørring)
-        return list
-    }
+    data class GrupperteAksjonspunktVenteårsak(
+        val system : Fagsystem,
+        val fagsakYtelseType: FagsakYtelseType,
+        val behandlingType: BehandlingType,
+        val aksjonspunktKode: String,
+        val frist: LocalDate,
+        val venteårsak: String,
+        val antall: Int
+    )
 
-    fun hentAllePåVent(): List<Oppgave> {
-        val startSpørring = System.currentTimeMillis()
-        val jsonOppgaver: List<String> = using(sessionOf(dataSource)) {
+    fun hentAllePåVentGruppert(): List<GrupperteAksjonspunktVenteårsak> {
```

Comment:
Trekk ut i egen repoklasse, gjerne i samme pakke som nøkkeltallklassen, slik at det ikke ligger sauset sammen med generell "logikk" for vedlikehold av køer og reservasjoner